dts/bindings/test/vnd,video-multi-port.yaml

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2024 tinyVision.ai Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Test Video device
+
+compatible: "vnd,video-multi-port"
+
+child-binding:
+  child-binding:
+    child-binding:
+      include: video-interfaces.yaml
+      properties:
+        reg:
+          type: array
```

Add a newline to the end of the file
```suggestion
          type: array

```

---

Will do, thanks !

---

done